documentation/api/query/v4/operators.markdown

```diff
@@ -180,8 +180,72 @@ Subqueries allow you to correlate data from multiple sources or multiple
 rows. For instance, a query such as "fetch the IP addresses of all nodes with
 `Class[Apache]`" would have to use both facts and resources to return a list of facts.
 
-Subqueries are unlike the other operators listed above. They always appear together
-in the following form:
+There are two forms of subqueries, implicit and explicit, and both forms work the
+same under the hood. The implicit form however, only requires you to specify the
+related entity, while the explicit form requires you to be specify exactly how
+data should be joined during the subquery.
+
+### `subquery` (Implicit Subqueries)
+
+*Note:* Implicit subqueries are a new experimental feature, be warned the functionality
+may change as we provide improvements.
+
+Implicit queries work like most operators, and simply require you to specify the
+related entity and the query to use:
+
+    ["subquery", "<ENTITY>", <SUBQUERY STATEMENT>]
+
+The entity is a mapping to the particular endpoint or data type you desire, however not
+all entities are supported from all endpoints, as not every relationship makes sense.
+Consult the endpoint document for details on what implicit relationships are supported:
+
+* [`catalogs`][catalogs]
+* [`edges`][edges]
+* [`environments`][environments]
+* [`events`][events]
+* [`facts`][facts]
+* [`fact_contents`][fact-contents]
+* [`fact_paths`][fact-paths]
+* [`nodes`][nodes]
+* [`reports`][reports]
+* [`resources`][resources]
+
+Internal to PuppetDB, we keep a mapping of how different entities relate to each
+other, and so no other data beyond the entity is needed in this case. This is
+different to explicit subqueries, for those you must specify yourself how
```

Comment:
'to', 'than' and 'from' are all valid forms of english, even in the United States. Seems like 'from' is a more acceptable international choice, so I'll switch it to that.

http://www.oxforddictionaries.com/words/different-from-than-or-to